app/validation/validator.py

```diff
@@ -2064,24 +2092,49 @@ def _get_answers_with_parent_ids(self, json_to_validate):
 
         return answers
 
-    def _get_questions_with_context(self, questionnaire_json):
+    @classmethod
+    def _get_answer_id_to_option_values_map(cls, json_to_validate):
```

Comment:
We should probably make this a class method and use using self.schema?

Reply:
I don't understand your comment. This is already a class method. `self.schema` isn't same as `json_to_validate`. Can you elaborate on what exactly you mean?

Reply:
I mean an instance method, self.schema is already held so it seems redundant to expect it to be passed.

Reply:
Unless I misunderstood, `self.schema` is `questionnaire_v1.json` and not the actual schema that is to be validated?

Reply:
I see, as you were then. Stand down.